core/chains/evm/log/mock_iLogPool_test.go

```diff
@@ -0,0 +1,117 @@
+// Code generated by mockery 2.9.4. DO NOT EDIT.
+
+package log
+
+import (
+	common "github.com/ethereum/go-ethereum/common"
+	mock "github.com/stretchr/testify/mock"
+
+	types "github.com/ethereum/go-ethereum/core/types"
+)
+
+// mockILogPool is an autogenerated mock type for the iLogPool type
+type mockILogPool struct {
```

Comment:
Did you end up using this?

Reply:
Not in this PR. But any other code which uses it(broadcast.go here) will use the mock to mock this out.
I added it here, so as to make it clear how the mock will look like, as per the guidelines.

Reply:
I'm finding it difficult to review this PR with unknown future context. Particularly why this kind of struct would be better mocked than e.g. faked or just manipulated directly to control state. Can we wait until we have usages to add this?

Reply:
If you mean wait adding this Mock until a later PR where it is used?
Yes, can do that.